stock_inventory_chatter/models/stock.py

```diff
@@ -8,9 +8,9 @@ class StockInventory(models.Model):
     _name = "stock.inventory"
     _inherit = ["stock.inventory", "mail.thread"]
 
-    state = fields.Selection(track_visibility="onchange")
-    location_ids = fields.Many2many(track_visibility="always")
-    start_empty = fields.Boolean(track_visibility="onchange")
+    state = fields.Selection(tracking=True)
+    location_ids = fields.Many2many(tracking=True)
+    start_empty = fields.Boolean(tracking=True)
```

Comment:
actually this field is not present in any view, so it does not make much sense tu track it.